### ISSUE TITLE
TabsAndIndentsVisitor aligns single line comments at column 0.

### ISSUE THREAD

**Issue by traceyyoshima** [openrewrite/rewrite#710]
In some cases, it's reasonable to aligned a single-line comment to the current column.

Based on the SAAS output, the majority of comments that originate at column 0 are from IntelliJ and should not be aligned.
It creates unnecessary noise in the PR, and prevents IntelliJ from quickly uncommenting the code.

This issue will add a quick condition to prevent single line comments from being aligned if indent == 0.

**Comment by traceyyoshima** [comment]
Looked into this and it's really complicated to adjust, since the list of comments in a Space may have various conditions and occur anywhere whitespace may exist.

The indent of the AST element is based on the suffix of the last comment, so the indent needs to happen on each suffix if the element is misaligned.

There may be more than one comment in the list, and the suffix of each comment sets the whitespace of the next comment.

Additionally, there are several inconsistent checks for new lines where space().getWhitespace().contains() only checks for a "\n" character. I think we should wait on this until issues are brought up from the community, and focus on things like consistent checks and expanding capabilities.

**Comment by traceyyoshima** [comment]
Closing, since the issue is on hold until a user voices a need for the changes.
